lib/iconv.js

```diff
@@ -47,7 +47,7 @@ function Iconv(fromEncoding, toEncoding)
 
   var conv = bindings.make(fixEncoding(fromEncoding),
                            fixEncoding(toEncoding));
-  if (conv === null) throw new Error('Conversion not supported.');
+  if (conv === null) throw new Error('Conversion from '+fromEncoding+' to '+toEncoding+' is not supported.');
```

Comment:
Can you wrap this at 80 columns and put spaces between the + operators?  Probably should be a `if (...) {` block now.